### PR TITLE
Add releases page

### PR DESCRIPTION
**Pull request by luisalima**
Adds releases page. For now, it only shows the latest release. When we have more releases, we will split them up, link to the downloads, and have a changelog for each product.

### CODE REVIEW

File: releases.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - All of these commands now support the --output flag and thus can report the result of the operation in machine-readable formats
+- Some commands now accept piped input, allowing for chaining of certain commands
+    - For example, `fyde-cli user list -q "John Doe" --list-all | fyde-cli user disable` can be used to disable all users that appear in the search results for "John Doe"
+    - For this to work, the piped command must output JSON (which is the default if no flags or configuration options are set).
+- Fixed bugs editing resources, where clearing the notes field would be impossible, and specifying port mappings is no longer mandatory every time a resource is edited
+- Improved handling of server error responses
+
+## Coming soon
+
+- Support for detection of antivirus, firewall and disk encryption status on Windows
```
Comment (oNaiPs) [inline releases.md]: @luisalima pls remove this before merging like we discussed

File: releases.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: Releases
+has_children: false
+nav_order: 9
+---
+# Releases
+
+## Fyde Proxy - Version 1.1.12
+
+Release date: 13-01-2020
```
Comment (pablosole) [inline releases.md]:
```suggestion
Release date: 15-01-2020
```

File: releases.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+### Improvements
+
+- Added support for TLS Session Tickets in cluster mode
```
Comment (pablosole) [inline releases.md]:
```suggestion
- Added support for TLS Session Tickets in cluster mode
- Added support for HTTP/S Proxies
```

**Comment by luisalima** [comment]
@pablosole @oNaiPs @GMartinez-Sisti can you take a look again please? I pushed the changes.

**Pull request merged**